uiowa/uids
# 3.x -  hide components

Resolves https://github.com/uiowa/uids/issues/898

# How to test

- Go to https://uids.brand.uiowa.edu/branches/3x_hide_components/ and compare against https://github.com/uiowa/uids/issues/898.  
- Compare against https://uids.brand.uiowa.edu/latest/ confirm that "Not implemented" options are still visible in https://uids.brand.uiowa.edu/branches/3x_hide_components/
